setup.cfg

```diff
@@ -132,6 +134,7 @@ filterwarnings =
     error
     ignore:unclosed <socket:ResourceWarning
     ignore:unclosed <ssl.SSLSocket:ResourceWarning
+    ignore:unclosed transport <asyncio.sslproto
```

Comment:
If this was happening only for one test, can we just ignore this one for that test instead of a global ignore like this?

Reply:
True!  I'll go ahead and see if I can replace it with a local `@pytest.mark.filterwarnings` decorator instead.